airbyte-cdk/python/unit_tests/sources/declarative/partition_routers/test_list_partition_router.py

```diff
@@ -37,29 +36,10 @@
 )
 def test_list_partition_router(test_name, partition_values, cursor_field, expected_slices):
     slicer = ListPartitionRouter(values=partition_values, cursor_field=cursor_field, config={}, parameters=parameters)
-    slices = [s for s in slicer.stream_slices(SyncMode.incremental, stream_state=None)]
+    slices = [s for s in slicer.stream_slices()]
     assert slices == expected_slices
 
 
-@pytest.mark.parametrize(
-    "test_name, stream_slice, last_record, expected_state",
-    [
-        ("test_update_cursor_no_state_no_record", {}, None, {}),
-        ("test_update_cursor_with_state_no_record", {"owner_resource": "customer"}, None, {"owner_resource": "customer"}),
-        ("test_update_cursor_value_not_in_list", {"owner_resource": "invalid"}, None, {}),
-    ],
-)
-def test_update_cursor(test_name, stream_slice, last_record, expected_state):
```

Comment:
Do we need to update any of these component-specific tests to replace tests for `update_cursor` with tests that hit the `Cursor.update_state` codepaths?

Reply:
PartitionRouters do not have the `Cursor` interface anymore which means they don't have `update_state` and `get_stream_state`. This makes sense as they don't have any state of their own. Does that make sense?

Reply:
Yep!